service/src/main/kotlin/no/nav/su/se/bakover/service/utbetaling/UtbetalingServiceImpl.kt

```diff
@@ -210,32 +220,35 @@ internal class UtbetalingServiceImpl(
                 fnr = sak.fnr,
                 utbetalinger = sak.utbetalinger,
                 behandler = saksbehandler,
+                stansDato = stansDato,
                 clock = clock,
             ).generate()
-        return simulerUtbetaling(utbetalingTilSimulering).mapLeft {
-            KunneIkkeStanseUtbetalinger.SimuleringAvStansFeilet
-        }.flatMap {
-            if (simulertStansHarBeløpUlikt0(it)) return KunneIkkeStanseUtbetalinger.SimulertStansHarBeløpUlikt0.left()
-            utbetal(it).mapLeft {
-                KunneIkkeStanseUtbetalinger.SendingAvUtebetalingTilOppdragFeilet
+        return simulerUtbetaling(utbetalingTilSimulering)
+            .mapLeft {
+                KunneIkkeStanseUtbetalinger.SimuleringAvStansFeilet
+            }.flatMap { simulertUtbetaling ->
+                kontollerSimulering(simulertUtbetaling)
+                    .mapLeft {
+                        when (it) {
+                            KontrollAvSimuleringFeilet.KunneIkkeTolkeSimulering -> KunneIkkeStanseUtbetalinger.KontrollAvSimuleringFeilet
```

Comment:
Gir det mening å legge på mer kontekst her for frontend? Eller må vi typ sjekke loggen uansett i de tilfellene vi får feil rundt simulering?

Reply:
I utgangspunktet er dette bare ekstra guards for situasjoner som ikke bør oppstå. I de tilfellene det eventuelt oppstår, må man gjerne gå ganske dypt i materien for å finne ut hva som er årsaken - i så måte heller jeg mot at vi uansett må inn å sjekke.